src/biogeochem/DUSTMod.F90

```diff
@@ -158,14 +204,100 @@ subroutine InitHistory(this, bounds)
          avgflag='A', long_name='turbulent deposition velocity 4', &
          ptr_patch=this%vlc_trb_4_patch, default='inactive')
 
+    this%dst_emiss_coeff_patch(begp:endp) = spval
+    call hist_addfld1d (fname='C_d', units='dimensionless',  &
```

Comment:
These should be broken up into ones that should be default on and ones that should be inactive. Ones that are default inactive can still be added by hand to a case in the namelist.

Reply:
I think they should all be default inactive, and added by hand as outputs if needed.

Reply:
I am unsure how to change them to by default inactive, so maybe I'll let you change that...

Reply:
Yes, I'll go ahead and do this. I just needed to know what should be default active. And since none need to be default active, this is easy to switch. They can then be easily turned on in the user_nl_clm by for example
adding the following to user_nl_clm

hist_fincl1 = 'C_d', 'WND_FRC_FT', 'WND_FRC_FT_DRY', 'WND_FRC_SOIL', ... 'DPFCT_ROCK'

Reply:
Indeed. thanks!

Reply:
This still needs to be done.

Reply:
@slevis-lmwg and I talked about this and agree this should be done.

I'll also add these output variables to the Leung specific tests so that they are exercised.

Reply:
OK, handled.